packages/graph-editor/src/components/flow/DropPanel/PanelItems.tsx

```diff
@@ -421,3 +421,11 @@ export const items = {
     },
   ],
 };
+
+export const addPanelItem = (item: PanelItem, category: string) => {
```

Comment:
This method pushes a new panel item for our custom node. We can use an existing category or add it inside a new one.